common/djangoapps/student/models.py

```diff
@@ -59,6 +59,26 @@
 AUDIT_LOG = logging.getLogger("audit")
 SessionStore = import_module(settings.SESSION_ENGINE).SessionStore  # pylint: disable=invalid-name
 
+UNENROLLED_TO_ALLOWEDTOENROLL = _('from unenrolled to allowed to enroll')
+ALLOWEDTOENROLL_TO_ENROLLED = _('from allowed to enroll to enrolled')
+ENROLLED_TO_ENROLLED = _('from enrolled to enrolled')
+ENROLLED_TO_UNENROLLED = _('from enrolled to unenrolled')
+UNENROLLED_TO_ENROLLED = _('from unenrolled to enrolled')
+ALLOWEDTOENROLL_TO_UNENROLLED = _('from allowed to enroll to enrolled')
+UNENROLLED_TO_UNENROLLED = _('from unenrolled to unenrolled')
```

Comment:
@chrisndodge we aren't translating [this line](https://github.com/edx/edx-platform/pull/8165/files#diff-9030096cb9659025eaf82ed406dcb128R423) which is also going into the DB, so I don't think these strings should be translated.